packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_GDSWCoarseOperator_def.hpp

```diff
@@ -356,206 +343,215 @@ namespace FROSch {
+                        << " | " << setw(41) << boolalpha << useVertexTranslations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "ShortEdges " << " | " << setw(19) << "Translations " << right
+                        << " | " << setw(41) << boolalpha << useShortEdgeTranslations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "ShortEdges " << " | " << setw(19) << "Rotations " << right
+                        << " | " << setw(41) << boolalpha << useShortEdgeRotations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "StraightEdges " << " | " << setw(19) << "Translations " << right
+                        << " | " << setw(41) << boolalpha << useStraightEdgeTranslations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "StraightEdges " << " | " << setw(19) << "Rotations " << right
+                        << " | " << setw(41) << boolalpha << useStraightEdgeRotations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "Edges " << " | " << setw(19) << "Translations " << right
+                        << " | " << setw(41) << boolalpha << useEdgeTranslations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "Edges " << " | " << setw(19) << "Rotations " << right
+                        << " | " << setw(41) << boolalpha << useEdgeRotations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "Faces " << " | " << setw(19) << "Translations " << right
+                        << " | " << setw(41) << boolalpha << useFaceTranslations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << "| " << left << setw(19) << "Faces " << " | " << setw(19) << "Rotations " << right
+                        << " | " << setw(41) << boolalpha << useFaceRotations << noboolalpha
+                        << " |"
+                        << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
+                        << setw(89) << "-----------------------------------------------------------------------------------------"
+                        << endl;
+                    }
+                }
```

Comment:
This is very difficult to compare side-by-side, but the code seems to be essentially the same. Or did you make any substantial change?

Reply:
I basically just moved some code into ``if (useForCoarseSpace)`` (in case some blocks were not used for coarse space).